enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/ConfigHolder.java

```diff
@@ -493,7 +492,7 @@ private void populateBackendJWKSConfiguration(JWTGenerator jwtGenerator) {
                 RSAKey jwk = new RSAKey.Builder(publicKey)
                         .keyUse(KeyUse.SIGNATURE)
                         .algorithm(getJWKSAlgorithm(jwtGenerator.getSigningAlgorithm()))
-                        .keyID(getJWKSThumbprint(cert))
+                        .keyID(generateThumbprint("SHA-256", cert, false))
```

Comment:
Let's import JWTUtil and invoke like JWTUtil.generateThumbprint

Reply:
ack